src/hooks.ts

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import { Context } from './context'
+
+export const useOpenAt = () => {
+  const { openAt } = React.useContext(Context)
```

Comment:
Name it `useGallery`, maybe? And expose an object `{ open }`. We can extend this API in future, with this. Want to note, that I own prefer `open` name for this method, because `at` meaning that you should pass an index, and it's already described in the signature (and can change over time, maybe).